core/tests/test_utils.py

```diff
@@ -809,9 +937,22 @@ def __get__(self, instance, owner):
         return self
 
     def pre_call_hook(self, args):
+        """Override this to do tasks that should be executed before the
+        actual function call.
+
+        Args:
+            args: Set of arguments that the function accepts.
+        """
         pass
 
     def post_call_hook(self, args, result):
+        """Override this to do tasks that should be executed after the
+        actual function call.
+
+        Args:
+            args: Set of arguments that the function accepts.
```

Comment:
Ditto --
```
    args: list(*). ...
    result: *. ...
```
(* means 'any type'). Please verify the above is correct, though.

Reply:
Done!